shared_model/bindings/CMakeLists.txt

```diff
@@ -18,16 +18,16 @@ if (SWIG_PYTHON OR SWIG_JAVA)
 
     set(CMAKE_SWIG_FLAGS "")
     include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-    set_source_files_properties(crypto.i PROPERTIES CPLUSPLUS ON)
+    set_source_files_properties(bindings.i PROPERTIES CPLUSPLUS ON)
 endif()
 
 if (SWIG_PYTHON)
     find_package(PythonLibs 2.7 REQUIRED)
     # path to where Python.h is found
     include_directories(${PYTHON_INCLUDE_DIRS})
 
-    swig_add_library(irohacrypto LANGUAGE python SOURCES crypto.i)
-    swig_link_libraries(irohacrypto ${Python_LIBRARIES} shared_ed25519_sha3)
+    swig_add_library(irohalib LANGUAGE python SOURCES bindings.i)
+    swig_link_libraries(irohalib ${Python_LIBRARIES} shared_model_proto_backend shared_model_ed25519_sha3 "-flat_namespace -undefined suppress")
```

Comment:
This flags cause:
```
clang-5.0: warning: argument unused during compilation: '-flat_namespace' [-Wunused-command-line-argument]
clang-5.0: warning: argument unused during compilation: '-undefined suppress' [-Wunused-command-line-argument]
```
And with -Werror will result error afterwards

Reply:
rename it to `iroha` or `libiroha`